nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/processors/script/ExecuteScriptGroovyTest.groovy

```diff
@@ -20,19 +20,17 @@ import org.apache.nifi.script.ScriptingComponentUtils
 import org.apache.nifi.util.MockFlowFile
 import org.apache.nifi.util.StopWatch
 import org.apache.nifi.util.TestRunners
-import org.junit.After
-import org.junit.Before
-import org.junit.BeforeClass
-import org.junit.Ignore
-import org.junit.Test
+import org.junit.*
```

Comment:
Please ensure you've run the Maven build with the `-Pcontrib-check` profile activated, it will catch checkstyle errors such as star imports like this. Thanks!

Reply:
Actually I don't think we have the Checkstyle rules for Groovy files in place, but just for consistency, it would be nice to avoid star imports where possible.

Reply:
You're right about it. Only the java code is checked. That's why I missed those imports. Will fix.

Reply:
fixed in 4614d29010d390d6ed5e50d56ec9fd3a29d55805